README.md

```diff
@@ -89,6 +91,25 @@ https://your-subdomain.imgix.net/images/demo.png?h=800&ar=3%3A2&fit=crop&dpr=5&s
 
 For more information to better understand `srcset`, we highly recommend [Eric Portis' "Srcset and sizes" article](https://ericportis.com/posts/2014/srcset-sizes/) which goes into depth about the subject.
 
+**Width Tolerance**
+
+The `srcset` width tolerance dictates the rate at which widths grow between entries in a srcset attribute. By default, this rate is set to 8 percent. This setting can be used to fine tune how many `srcset` pairs are generated when invoking `Imgix::Path#to_srcset`. Users can specify their own width tolerance by passing a positive numeric value using the `width_tolerance` keyword argument:
```

Comment:
Two things here:

1. I don't feel like we're actually explaining the right thing here. 8% isn't the growth rate, it's the maximum allowable size difference between the *downloaded* image and the *rendered* image.
2. We should explain that we use 8% by default because we've found that's the largest value we can use without any noticeable visual degradation of the image.

```suggestion
The `srcset` width tolerance dictates the maximum tolerated size difference between an image's downloaded size and its rendered size. For example: setting this value to 0.1 means that an image will not render more than 10% larger or smaller than its native size. In practice, the image URLs generated for a width-based srcset attribute will grow by twice this rate. A lower tolerance means images will render closer to their native size (thereby reducing rendering artifacts), but a large srcset list will be generated and consequently users may experience lower rates of cache-hit for pre-rendered images on your site.

By default this rate is set to 8 percent, which we consider to be the ideal rate for maximizing cache hits without sacrificing visual quality. Users can specify their own width tolerance by passing a positive numeric value using the `width_tolerance` keyword argument:
```

Reply:
Well said 👍